db/src/main/resources/db/migration/V1_059__partial_unique_index_nino_claim_table.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX unique_nino_for_active_claim ON claim (nino) WHERE (claim_status in ('NEW', 'ACTIVE', 'PENDING', 'PENDING_EXPIRY'));
```

Comment:
Minor, but unnecessary brackets around `claim_status in ('NEW', 'ACTIVE', 'PENDING', 'PENDING_EXPIRY')`